tests/helpers/metadata.py

```diff
@@ -56,7 +57,10 @@ def _find_monitor(monitors, mon_type):
 def _get_monitor_metrics(monitor, included):
     for metric, info in (monitor.get("metrics") or {}).items():
         if info["included"] == included:
-            yield metric
+            if sys.platform == "win32" and "linux only" not in info["description"].lower():
```

Comment:
👎 This needs to be a dedicated field in the metadata.yaml that gets passed through.